ci-operator/step-registry/rosa/operator/e2e/rosa-operator-e2e-commands.sh

```
@@ -0,0 +1,65 @@
+log "Connected to cluster: $(oc whoami --show-server)"
+
+# Run the operator e2e tests
+JUNIT_REPORT="${ARTIFACT_DIR}/junit-${OPERATOR_NAME}-e2e.xml"
+GINKGO_FLAGS="--ginkgo.junit-report=${JUNIT_REPORT} --ginkgo.v"
+
+if [[ -n "${GINKGO_LABEL_FILTER:-}" ]]; then
+    GINKGO_FLAGS="${GINKGO_FLAGS} --ginkgo.label-filter=${GINKGO_LABEL_FILTER}"
+fi
+
+log "Running ${OPERATOR_NAME} e2e tests..."
+/usr/local/bin/e2e.test ${GINKGO_FLAGS} || {
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)